specification/iotsecurity/resource-manager/Microsoft.IoTSecurity/preview/2021-02-01-preview/sites.json

```diff
@@ -0,0 +1,266 @@
+              "$ref": "#/definitions/SitesModel"
+            }
+          },
+          "201": {
+            "description": "Created IoT site",
+            "schema": {
+              "$ref": "#/definitions/SitesModel"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete IoT site": {
+            "$ref": "./examples/Sites/Delete.json"
+          }
+        },
+        "tags": [
+          "Sites"
+        ],
+        "operationId": "Sites_Delete",
+        "description": "Delete IoT site",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SiteScope"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted IoT site"
+          },
+          "204": {
+            "description": "IoT site does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "SitesList": {
+      "description": "List of IoT sites",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List data",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SitesModel"
+          }
+        }
+      }
+    },
+    "SitesModel": {
+      "type": "object",
+      "description": "IoT site model",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "IoT site properties",
+          "$ref": "#/definitions/SiteProperties"
+        },
+        "systemData": {
+          "readOnly": true,
+          "type": "object",
+          "description": "Azure Resource Manager metadata containing createdBy and modifiedBy information.",
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "SiteProperties": {
+      "type": "object",
+      "description": "IoT site properties",
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "Display name of the IoT site"
+        },
+        "tags": {
```

Comment:
What is this representing?  This directly conflicts with the concept of resource manager tags.  There should be some first class way to indicate to the user that this is different

Reply:
Note: please file an issue to fix this in the next api-version.  Since you are carrying this over from a previous namespace, and this is a preview, this can be allowed for now, but generally, having two properties named tags is confusing.